python/cugraph/tests/test_symmetrize.py

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+from itertools import product
+
+import pytest
+
+import cudf
+import cugraph
+from cugraph.tests import utils
+from librmm_cffi import librmm as rmm
+from librmm_cffi import librmm_config as rmm_cfg
+
+
+def test_version():
+    gc.collect()
+    cugraph.__version__
+
+
+DATASETS = ['../datasets/karate',
```

Comment:
While `karate.csv` is an undirected graph, the version we have in the repo already stores an undirected edge as a directed edge in both directions (notice the 78 undirected edges in the `.mtx` vs. 156 lines in the `.csv` file). So, the symmetrize feature should not do anything on this dataset.